docs/source/quick_start/first_run.rst

```diff
@@ -0,0 +1,168 @@
+
+First Run
+=========
+
+Now, you have installed MCDR, it's time to run it for the first time
+
+.. note::
+
+    The most common use of MCDR is to control a Minecraft server, so we also take this as the demonstrated use case here
+
+.. tip::
+
+    To manage a Minecraft server with MCDR, it is recommended to have basic knowledge of common Minecraft servers
+    
+    Which means, you should know:
+
+    * The Minecraft servers: what they are, how to configure (one of) them
+    * Start command of Minecraft server: what it is, what its arguments mean
+    * Console of the server: what it prints, what command can you send to it
+    * How to manage the server: configure, mods / plugins etc.
+    
+    If you have not, Google is your friend
+
+Prepare
+-------
+
+To manage a Minecraft server with MCDR, you should have a **well-configured** Minecraft server
+
+For example, this is the typical directory structure of a vanilla Minecraft server:
+
+.. code-block:: text
+
+    server/
+    ├── libraries/
+    ├── logs/
+    ├── versions/
+    ├── world/
+    ├── banned-ips.json
+    ├── banned-players.json
+    ├── eula.txt
+    ├── ops.json
+    ├── minecraft_server.jar
+    ├── server.properties
+    ├── usercache.json
+    └── whitelist.json
+
+With this command to start it:
+
+.. code-block:: bash
+
```

Comment:
Append a `cd server` inside the code block

Reply:
maybe change the directory name to `.`  instead of adding the cd command

Reply:
up to you